Repo/TodoRepo.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.AspNetCore.Mvc;
+using todoDotNet6.Repo;
+
+namespace todoDotNet6.TodoRepo
+{
+
+    public interface ITodoRepository : IRepository<Todo,Guid> {}
+
+    public class TodoRepo : ControllerBase, ITodoRepository
+    {
+        public static Dictionary<Guid, Todo> todos = new Dictionary<Guid, Todo>() {};
```

Comment:
If you arent adding items with a collection initializer then `{}` is not necessary